src/plugins/data/public/search/aggs/buckets/filters.ts

```diff
@@ -47,6 +48,13 @@ export interface FiltersBucketAggDependencies {
   getInternalStartServices: GetInternalStartServicesFn;
 }
 
+export interface AggParamsFilters extends BaseAggParams {
+  filters?: Array<{
+    input: Query;
+    label: string;
+  }>;
+}
```

Comment:
To make this interface accurate, don't we need to override the `customLabels` from `BaseAggParams` here, since `AggParamsFilters` explicitly sets it to `false` and therefore it isn't a valid param at all?